vector/src/main/AndroidManifest.xml

```diff
@@ -395,6 +395,13 @@
             android:foregroundServiceType="mediaProjection"
             tools:targetApi="Q" />
 
+        <service
+            android:name=".features.call.audio.MicrophoneAccessService"
+            android:exported="false"
+            android:foregroundServiceType="microphone"
+            android:permission="android.permission.FOREGROUND_SERVICE_MICROPHONE">
+        </service>
+
```

Comment:
I think you need to declare the permission as well. You can add around line 56:

```xml
    <!-- For MicrophoneAccessService -->
    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MICROPHONE" />
```

It is working since this permission comes with the Jitsi library, but better to explicitely add it in our manifest.

Reply:
did it and pushed, tell me if there is any more comments of improvement that need to be done :)